### PR TITLE
add cytominer-database

### PR DESCRIPTION
**Pull request by gwaygenomics**
@conda-forge/help-python

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there (👋)

### CODE REVIEW

File: recipes/cytominer_database/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f253822e2e9c01e815492e06a6611405c4bb8daa615f24745c4308f41645ec10"
+
+build:
```
Comment (beckermr) [inline recipes/cytominer_database/meta.yaml]: This recipe may be able to be made `noarch: python`. Can you check the list in the docs?

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Comment (gwaygenomics) [reply recipes/cytominer_database/meta.yaml]: looks like it is able, I will fix

**Comment by gwaygenomics** [comment]
Thanks for the suggestion @beckermr ! It looks like all checks have passed. Good to merge?

**Comment by isuruf** [comment]
Can you create a PR or issue upstream to package the license file in the sdist?

**Comment by gwaygenomics** [comment]
> We have to ship the license file. This is required by BSD. Once we add that back we are good to go.

Added and checks passed. Anything else need addressing? Thanks again!

**Comment by beckermr** [comment]
Great! See @isuruf's comment above. Merging in the meantime.

**Comment by gwaygenomics** [comment]
> Can you create a PR or issue upstream to package the license file in the sdist?

> Great! See @isuruf's comment above. Merging in the meantime.

I am not sure what this means (Thanks for the merge!)

**Comment by beckermr** [comment]
So to get the license file in the sdist on pypi, you need to have a MANIFEST.in in the package repo with the license file included there. @isuruf means to make an issue on the upstream repo requesting that this be done.

**Comment by gwaygenomics** [comment]
> So to get the license file in the sdist on pypi, you need to have a MANIFEST.in in the package repo with the license file included there.

Got it!

**Pull request merged**